msticpy/data/queries/kql_mdatp_alerts.yaml

```diff
@@ -1,39 +1,39 @@
 metadata:
   version: 1
   description: MDATP Queries
-  data_environments: [MDATP]
+  data_environments: [MDATP, MDE, MD365]
```

Comment:
Given the product is Microsoft 365 Defender so we want to make this M365D?

Reply:
Yep - changed all to M365D